sunpy/map/maputils.py

```diff
@@ -581,6 +586,9 @@ def pixelate_coord_path(smap, coord_path, *, bresenham=False):
     If a pixel intersects the coordinate path at only its corner, it may not be
     returned due to the limitations of floating-point comparisons.
 
+    If the coordinate path exceeds the bounds of the map, those returned pixel
+    coordinates will not correspond to actual pixels with data.
```

Comment:
```suggestion
    If the coordinate path is at least partly outside the edges of the map an error is raised.
```

Reply:
@hayesla how about my above suggestion?

Reply:
Your suggestion is not correct, as the point of this note is that an error is *not* raised in this situation

Reply:
```suggestion
    If the coordinate path crosses pixels outside the bounds of the map, the returned pixel
    coordinates will not correspond to actual pixels with data.
```
Ah, how about this instead?